### ISSUE TITLE
Scrobbling not working (at least with Google music and Youtube)

### ISSUE THREAD

**Issue by rhopp** [david-sabata/web-scrobbler#717]
My Last.FM scrobbling stopped working around beginning of june. I mainly scrobble from google music and youtube.
When I'm inspecting bacground page,  scrobbler recognizes the song which is being played (L.FM response with song,info etc. is in console output), but probabbly when it tries to scrobble that song, I get:
```
GET https://ws.audioscrobbler.com/2.0/?method=auth.getsession&api_key=d9bb...31ba8&api_sig=af4124...6647fe837&format=json 403 (Forbidden)

  l.cors.a.crossDomain.send @ jquery-2.1.0.min.js:4
  o.extend.ajax @ jquery-2.1.0.min.js:4
  o.(anonymous function) @ jquery-2.1.0.min.js:4
  o.extend.getJSON @ jquery-2.1.0.min.js:4
  tradeTokenForSession @ lastfm.js:133
  (anonymous function) @ lastfm.js:91
  (anonymous function) @ chromeStorage.js:26
  target.(anonymous function) @ extensions::SafeBuiltins:19
  safeCallbackApply @ extensions::sendRequest:21
  handleResponse @ extensions::sendRequest:72
```
and
```
auth.getSession failed: Forbidden, error
  (anonymous function) @ lastfm.js:143
  j @ jquery-2.1.0.min.js:2
  k.fireWith @ jquery-2.1.0.min.js:2
  x @ jquery-2.1.0.min.js:4
  (anonymous function) @ jquery-2.1.0.min.js:4
```

I thougth that this could be authentication problem, so I clicked on "Reauthenticate with L.FM" on preferences page of scrobbler. In console is just 
```
getToken response:
<lfm status="ok">
   <token>8349...10218</token>
</lfm>
```
So I guess the reauthentication was succesfull, but the scrobbling is still not working.

**Comment by bes-internal** [comment]
after reauthentication got error at core/background/services/last.fm:92 : session.trim() not a func
after reload extension i always get error as in #713

**Comment by daberkow** [comment]
Tested @janbaykara fix, it works, I had to reauth first but then its good to go.

**Comment by trs-mtn** [comment]
@janbaykara I switched off extension and after editing files, chrome said that the extension is modified and asking for repair. 
When repaired the files was automaticlly changed with origins from 1.51. I will try later when the chrome is closed.

**Comment by bes-internal** [comment]
@trs-mtn wait for the next release or 
1. Download and unpack zip from https://github.com/janbaykara/web-scrobbler/tree/auth-fix
2. Open chrome://extensions/ and check the Developer mode checkbox on the top right corner. 
3. Load unpacked extension

**Comment by david-sabata** [comment]
Should be resolved by #718
